src/routes/controllers/articles.controller.js

```diff
@@ -50,9 +51,54 @@ export default async function create(req, res) {
       res,
       201,
       'your article has been created successfully',
-      createArticle,
+      createdArticle,
     );
   } catch (error) {
-    return errorResponse(res, 401, error.message);
+    return errorResponse(res, 500, error.message);
+  }
+}
+
+/**
+ * @description users can delete articles there have created.
+ * @param {object} req
+ * @param {object} res
+ * @returns {object} response object
+ */
+export async function removeArticle(req, res) {
+  try {
+    const userInfo = await verifyToken(
+      req.headers['x-access-token'] || req.headers.authorization,
+    );
+    const getArticle = await Article.findOne({
+      where: { id: req.params.article_id },
+    });
+
+    if (getArticle) {
+      if (getArticle.userId === userInfo.id) {
+        const deletedArticle = await Article.destroy({
+          where: { id: req.params.article_id },
+        });
+        if (deletedArticle > 0) {
```

Comment:
why not do `if (deletedArticle)` instead of checking if its greater than 0.

Reply:
`deleteArticle` returns a number. so I am getting number above 0. 
it does not render to `false, null or undefined` because the variable is an actual number.